client/src/plus/common/login-banner.scss

```diff
@@ -0,0 +1,41 @@
+@use "../../ui/vars" as *;
+
+.login-banner {
+  background-color: var(--background-success);
+  border: 2px solid var(--background-success);
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+
+  .plus-link {
+    color: var(--text-primary);
+    font-style: normal;
+    text-decoration: underline;
+    text-decoration-thickness: 0.15rem;
+    text-underline-offset: 0.1em;
+  }
+
+  @media screen and (min-width: $screen-md) and (max-width: ($screen-lg - 1)) {
+    > span {
+      /* Show text (in two lines) on the left, while "Log in" / "Sign up" are (in one line) on the right. */
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  @media screen and (max-width: ($screen-lg - 1)) {
+    .auth-container {
+      /* Always keep "Log in" / "Sign up" (in one line) in bottom-right corner. */
+      align-self: end;
+      flex-flow: row;
+      justify-content: flex-end;
+      margin-bottom: 0;
+    }
+  }
+
+  @media screen and (max-width: ($screen-md - 1)) {
+    /* Show text (in one line) above "Log in" / "Sign up" (in one line). */
+    flex-direction: column;
+    gap: 0.5rem;
+  }
+}
```

Comment:
This is way too complex just to make the button wrap nicely on small screens.

Reply:
fixed in https://github.com/mdn/yari/pull/9357/commits/304bcd4ef55474eb5aea7e2f0f4ce368eb36a858

button now wraps like so:

![Screen Shot 2023-07-25 at 17 48 49](https://github.com/mdn/yari/assets/755354/581d3263-b015-4863-a7a1-b641af4e07c6)